test/modules/TestResultCoordinator/Startup.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.DependencyInjection;
+
+    public class Startup
```

Comment:
We might want to consolidate the Startup classes in TestAnalyzer and here into one class living somewhere shared like ModuleLib

Reply:
Not for now; and TestAnalyzer should be removed in the near future.

Reply:
Since TestAnalyzer is going away, we won't consolidate right now